docs/content/about-iota/iota-architecture/consensus.mdx

```diff
@@ -5,15 +5,51 @@ description: Overview of the IOTA consensus.
+- Energy efficiency: Unlike Proof-of-Work (PoW), Mysticeti's consensus mechanism requires minimal computational resources.
+- Security & decentralization: The protocol resists common attacks by relying on an honest majority of validators while maintaining a decentralized structure.
 
-For more details including correctness proofs, the [Mysticeti paper](https://arxiv.org/pdf/2310.14821) is the best source.
+## Delegated Proof-of-Stake (dPoS) in IOTA
+
+IOTA employs Delegated Proof-of-Stake (dPoS) as the overarching consensus mechanism. In this model:
+
+- Token holders delegate their voting power to validators.
+- Validators participate in block production and execute the Mysticeti protocol to finalize transactions.
+- Staking rewards serve as the main incentive for validators to behave honestly and secure the network.
+- This structure ensures security and decentralization while maintaining scalability.
+
+## Consensus Committee
+
+Validators that participate in the consensus protocol collectively form the **Consensus Committee**. This committee is responsible for executing the consensus mechanism and finalizing transactions.
+
+### Key Aspects of the Consensus Committee
+
+- **Validator Set**: The consensus committee consists of validators, which are selected based on delegated stake.
+- **Fixed Over an Epoch**: The validator set remains unchanged during an epoch to ensure stability.
+- **Roles of Validators**:
+  - Accept transactions and execute the consensus protocol.
+  - Maintain a consistently ordered ledger state.
+  - Prevent double-spending and conflicts.
+
+For more details on validator staking, refer to the [Validators Staking](../tokenomics/validators-staking.mdx) page. To understand epoch transitions, see the [Epochs](./epochs.mdx) documentation.
+
+## The Mysticeti Protocol
+
+The Mysticeti protocol is a leaderless BFT protocol optimized for low-latency and high-throughput using an uncertified DAG structure. It is designed to facilitate efficient and parallel transaction processing while maintaining strong security guarantees.
+
+**Mysticeti** is a **Byzantine Fault Tolerant (BFT) consensus protocol** optimized for **low latency** and **high throughput**. It operates on a **Directed Acyclic Graph (DAG)** structure and ensures finality in a short time.
+
+### Key Features of Mysticeti
+
+- **Parallel Block Proposals**: Multiple validators can propose blocks simultaneously, maximizing network bandwidth and ensuring censorship resistance. This is a core advantage of DAG-based protocols.
+- **Three-Round Commit Rule**: Transactions achieve finality in just three rounds of messages, matching the efficiency of pBFT and reaching the theoretical minimum for BFT consensus.
+- **Optimized Leaderless Voting**: Unlike traditional leader-based protocols, Mysticeti allows validators to vote on blocks and certify them in parallel, significantly reducing median and tail latencies.
```

Comment:
Not sure where the leaderless is coming from. But I mean afaik a leader is chosen if I remember the video from Alberto
```suggestion
- **Optimized Voting**: Unlike traditional leader-based protocols, Mysticeti allows validators to vote on blocks and certify them in parallel, significantly reducing median and tail latencies.
```